### PR TITLE
pkg/manifests/telemeter: bump size limit to 5mb

### PR DESCRIPTION
**Pull request by s-urbaniak**
/cc @openshift/openshift-team-monitoring

### CODE REVIEW

File: pkg/manifests/manifests.go [hunk 1]
```diff
@@ -2591,6 +2591,7 @@ func (f *Factory) TelemeterClientDeployment(proxyCABundleCM *v1.ConfigMap) (*app
 	for _, m := range f.config.PrometheusK8sConfig.TelemetryMatches {
 		cmd = append(cmd, fmt.Sprintf("--match=%s", m))
 	}
+	cmd = append(cmd, "--limit-bytes=409600")
```
Comment (lilic) [inline pkg/manifests/manifests.go]: @brancz suggested a higher limit.

/hold

Comment (brancz) [inline pkg/manifests/manifests.go]: let's bump this immediately to 5mb

Comment (s-urbaniak) [reply pkg/manifests/manifests.go]: done

**Comment by s-urbaniak** [comment]
ptal @openshift/openshift-team-monitoring

**Comment by lilic** [comment]
/cherry-pick release-4.3

just so we have it ready :)

**Comment by s-urbaniak** [comment]
i think the cherry-pick will not be super clean, but yes, we need it.

**Pull request merged**